src/ScriptTools.sol

```diff
@@ -37,6 +37,18 @@ library ScriptTools {
         string memory chainInputFolder = string(abi.encodePacked("/script/input/", vm.toString(block.chainid), "/"));
         return vm.readFile(string(abi.encodePacked(root, chainInputFolder, input, ".json")));
     }
+    
+    /**
+     * @notice Use standard environment variables to load config.
+     * @dev Will first check FOUNDRY_SCRIPT_CONFIG_TEXT for raw json text.
+     *      Falls back to FOUNDRY_SCRIPT_CONFIG for a standard file definition.
+     */
+    function loadConfig() internal returns (string memory config) {
+        config = vm.envOr("FOUNDRY_SCRIPT_CONFIG_TEXT", string(""));
+        if (eq(config, "")) {
+            config = readInput(vm.envString("FOUNDRY_SCRIPT_CONFIG"));
```

Comment:
Should we pass a string for a second fallback?

Reply:
Like `ScriptTools.loadConfig("fallback")` which would attempt to load the json file "fallback.json" if no env var? If so then yes makes sense to me.

Reply:
Anyways. I've added it.

Reply:
Yeah exactly this, but I'm not seeing it

Reply:
Probably missing the push

Reply:
ok should be good now